src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -1230,7 +1230,7 @@ class SymbolTableVisitor : public CommonVisitor<SymbolTableVisitor> {
                         current_procedure_abi_type = ASR::abiType::BindC;
                     } else if (name == "overload") {
                         overload = true;
-                    } else {
+                    } else if (name != "interface") {
```

Comment:
I think this is a cleaner way to do this:
```suggestion
                    } else if (name == "interface") {
                        // TODO: Implement @interface
                    } else {                    
```